BREAKING.md

```diff
@@ -15,9 +15,121 @@ It's important to communicate breaking changes to our stakeholders. To write a g
+
+### Container-loader deprecation removals
+
+The following types in the @fluidframework/container-loader package are not used by, or necessary to use our public api, so have been removed from export in this release:
+
+-   IContainerLoadOptions
+-   IContainerConfig
+-   IPendingContainerState
+-   ISerializableBlobContents
+
+### Closing Container no longer disposes
+
+Calling `IContainer.close(...)` will no longer dispose the container runtime, document service, or document storage service.
+
+If the container is not expected to be used after the `close(...)` call, replace it instead with a `IContainer.dispose(...)` call (this should be the most common case). Using `IContainer.dispose(...)` will no longer switch the container to "readonly" mode and relevant code should instead listen to the Container's "disposed" event.
+If you intend to pass your own critical error to the container, use `IContainer.close(...)`. Once you are done using the container, call `IContainer.dispose(...)`.
+
+Please see the [Closure](packages/loader/container-loader/README.md#Closure) section of Loader README.md for more details.
+
+### `IContainer.dispose` is now required
+
+`IContainer.dispose` is now a required method. This method should dispose any resources and switch the container to a permanently disconnected state.
+
+Please see the [Closure](packages/loader/container-loader/README.md#Closure) section of Loader README.md for more details.
+
+### ISummarizerRuntime on/off op required
+
+The `on("op")` and `off("op")` methods on `ISummarizerRuntime` are now required. These listener methods are needed to accurately run summary heuristics.
+
+### Driver param removed from appendToMergeTreeDeltaRevertibles
+
+The first parameter, driver, of the function appendToMergeTreeDeltaRevertibles has been removed. Additionally, the interface MergeTreeRevertibleDriver has been simplified, and no longer requires:
+
+-   createLocalReferencePosition
+-   localReferencePositionToPosition
+-   getPosition
+-   getContainingSegment
+
+### PureDataObject.getFluidObjectFromDirectory removed
+
+`PureDataObject.getFluidObjectFromDirectory` was deprecated in a previous release and has been removed.
+
+### IProvideContainerRuntime and IContainerRuntime member removed
+
+`IProvideContainerRuntime` and its `IContainerRuntime` member were deprecated in a previous release and have been removed. This applies to the `ContainerRuntime` class as well.
+
+### IntervalCollection removed
+
+The exports deprecated in [IntervalCollection public export deprecated](#intervalCollection-public-export-deprecated) have been removed.
+
 # 2.0.0-internal.4.4.0
 
-## 2.0.0-internal.4.3.0 Upcoming changes
+## 2.0.0-internal.4.4.0 Upcoming changes
```

Comment:
Given that this branch is on version internal.5 now, .4.4.0 changes shouldn't be listed as upcoming.